python/lsst/pipe/base/argumentParser.py

```diff
@@ -513,8 +520,6 @@ def _parseDirectories(self, namespace):
         # be run as there may be an environment variable set for the output
         else:
             namespace.output = _fixPath(DEFAULT_OUTPUT_NAME, namespace.rawOutput)
```

Comment:
I suggest removing this else clause. $PIPE_OUTPUT_ROOT was always intended to be a parent dir, never the actual output dir. This bug or misfeature has been present for a long time, but it should be squashed.